tests/std/tests/P0896R4_views_filter/test.cpp

```diff
@@ -347,13 +347,6 @@ int main() {
         test_one(lst, only_even_ints);
     }
 
-    // Validate a non-view borrowed range
-    {
-        constexpr span s{some_ints};
-        STATIC_ASSERT(test_one(s, only_even_ints));
-        test_one(s, only_even_ints);
-    }
-

```

Comment:
do we have coverage for this? Do such pathological ranges still exist (even though span isn't one)?

Reply:
I mentioned this briefly in the log comment: I think non-view borrowed ranges are now rare enough (I literally do not know of any non-synthetic examples) not to deserve specific testing. Shout if you think we should have a synthetic test case, and I'll add one - but preferably in a separate PR.